src/shogun/lib/parameter_observers/ObservedValue.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Giovanni De Toni
+ *
+ */
+
+#include <shogun/base/Parameter.h>
+#include <shogun/lib/parameter_observers/ObservedValue.h>
+
+using namespace shogun;
+
+ObservedValue::ObservedValue(int64_t step, std::string name)
+    : CSGObject(), m_step(step), m_name(name)
+{
+	SG_ADD(&m_step, "step", "Step", ParameterProperties::NONE);
+	this->watch_param(
+	    "name", &m_name,
+	    AnyParameterProperties(
+	        "Name of the observed value", ParameterProperties::NONE));
```

Comment:
Isn’t NONE the default? If not shouldn’t it? @gf712

Reply:
Yes, it is indeed the default value.